### PR TITLE
Fix more typos

### PR DESCRIPTION
**Pull request by szepeviktor**
Found more misspellings.

These typos are crazy. Please help me with more commits as necessary.

### CODE REVIEW

**Comment by liamwhite** [comment]
Here's one I noticed myself today if you want to include it:
https://github.com/yuzu-emu/yuzu/blob/c683ec2bcb71db6db504d4f1161a7830a8c42774/src/video_core/renderer_vulkan/vk_scheduler.h#L58

File: src/android/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -410,7 +410,7 @@
     <string name="language_japanese" translatable="false">日本語</string>
     <string name="language_english" translatable="false">English</string>
     <string name="language_french" translatable="false">Français</string>
-    <string name="langauge_german" translatable="false">Deutsch</string>
+    <string name="language_german" translatable="false">Deutsch</string>
```
Comment (liamwhite) [inline src/android/app/src/main/res/values/strings.xml:413]: This also needs to be updated in arrays.xml
https://github.com/yuzu-emu/yuzu/blob/e8671ed04e53ca099d8eea06023b002be52395b4/src/android/app/src/main/res/values/arrays.xml#L32

Comment (szepeviktor) [reply src/android/app/src/main/res/values/strings.xml:413]: Done 🍏

**Review by FearlessTobi — APPROVED** [review]

**Comment by szepeviktor** [comment]
After merging this PR a typo in the brand new `npad_vibration.cpp` needs to be fixed
https://github.com/yuzu-emu/yuzu/blob/c683ec2bcb71db6db504d4f1161a7830a8c42774/src/hid_core/resources/npad/npad_vibration.cpp#L33

**Comment by german77** [comment]
I can't say I'm not guilty. This PR hasn't been merged. You can still add commits to fix CI.

**Comment by szepeviktor** [comment]
> You can still add commits to fix CI.

Done.

**Comment by szepeviktor** [comment]
@liamwhite Thank you for merging.

Please consider using `typos` instead of `codespell`.

**Pull request merged**